### PR TITLE
KEDA-2021-Annual Review

### PR DESCRIPTION
**Pull request by tomkerkhove**
Signed-off-by: Tom Kerkhove <kerkhove.tom@gmail.com>

Our annual review for KEDA.

Closes https://github.com/kedacore/governance/issues/13

### CODE REVIEW

File: reviews/2021-KEDA-annual.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+| Maintainer           | GitHub ID                                     | Affiliation |
+| -------------------- | --------------------------------------------- | ----------- |
+| Zbynek Roubalik      | [zroubalik](https://github.com/zroubalik)     | Red Hat     |
+| Tom Kerkhove         | [tomkerkhove](https://github.com/tomkerkhove) | Codit       |
+| Jeff Hollan          | [jeffhollan](https://github.com/jeffhollan)   | Microsoft   |
+| Anirudh Garg         | [anirudhgarg](https://github.com/anirudhgarg) | Microsoft   |
+| Ahmed ElSayed        | [ahmelsayed](https://github.com/ahmelsayed)   | Microsoft   |
+
+These have not changed since we have joined CNCF, but this is about to change soon.
```
Comment (dzolotusky) [inline reviews/2021-KEDA-annual.md:39]: Can you provide some more context for "this is about to change soon"?

Comment (tomkerkhove) [reply reviews/2021-KEDA-annual.md:39]: https://github.com/kedacore/governance/issues/17

**Comment by tomkerkhove** [comment]
Great news @amye! Do we need to do anything else?

**Comment by tomkerkhove** [comment]
We're good to go, thanks a ton!

**Pull request merged**